crates/node-e2e-tests/tests/it/eth.rs

```diff
@@ -116,6 +116,97 @@ async fn can_run_eth_node() -> eyre::Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn can_run_eth_node_with_auth_engina_api_over_ipc() -> eyre::Result<()> {
+    let tasks = TaskManager::current();
+    let test_suite = TestSuite::new();
+
+    // Node setup
+    let node_config = NodeConfig::test()
+        .with_chain(test_suite.chain_spec.clone())
+        .with_rpc(RpcServerArgs::default().with_http());
```

Comment:
I don't see any ipc settings here?

why does ipc work here?
this should always be opt on, only if auth_ipc i set